src/contributing/adding_new_operation_how_to.md

```diff
@@ -0,0 +1,108 @@
+# Adding operations to Parsec
+
+This mini how-to will cover how to add operations to the backend and frontend of Parsec. The steps
+for adding operations to a backend will differ between providers, this guide will cover Mbed Crypto.
+The steps for adding operations to the frontend should be the same for all backend providers.
+
+## Backend (Mbed Crypto)
+
+### psa-crypto-sys
+
+Locate the parts of the PSA API in `mbedtls` that will need to be used during the operation you are
```

Comment:
I think you can replace `mbedtls` with "Mbed Crypto" as it is the actual component implementing the PSA APIs, even if contained in the Mbed TLS project.